packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -70,13 +70,8 @@ export class VaAccordion {
 
   @Listen('accordionItemToggled')
   itemToggledHandler(event: CustomEvent) {
-    // The event target is the button, and it has a parent which is a header.
-    // It is the parent of the header (the root item) that we need to access
-    // The final parentNode will be a shadowRoot, and from there we get the host.
-    // https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot/host
-    const clickedItem =
-      event.detail.currentTarget.parentNode.parentNode.host || // if we are on IE, `.host` won't be there
-      event.detail.currentTarget.parentNode.parentNode;
+    // eslint-disable-next-line i18next/no-literal-string
+    const clickedItem = (event.target as HTMLElement).closest('va-accordion-item');
```

Comment:
Is this essentially just an IE cleanup/refactor?

Reply:
Basically. On that topic, what browsers do we need to support?

Reply:
> that browsers do we need to support?

I forgot to answer your actual question! Here is our current [browser support policy](https://vfs.atlassian.net/wiki/spaces/pilot/pages/1847951366/Browser+Support+Policy):

![Screenshot 2023-04-20 at 10 10 53 AM](https://user-images.githubusercontent.com/872479/233409503-e4d8fdb5-a958-47e8-b9e8-963c5d16c508.png)